getting_started/introduction/first_look_at_the_editor.rst

```diff
@@ -0,0 +1,155 @@
+
+.. _doc_intro_to_the_editor_interface:
+
+First look at Godot's editor
+============================
+
+This page will give you a brief overview of Godot's interface. We're going to
+look at the different workspaces and docks to help you situate yourself.
+
+.. seealso:: For a comprehensive breakdown of the editor's interface and how to
+             use it, see the :ref:`Editor manual <toc-learn-editor>`.
+
+The Project manager
+-------------------
+
+When you launch Godot, the first window you see is the Project Manager. In the
+default tab, "Projects," you can manage existing projects, import or create new
+ones, and more.
+
+.. image:: img/editor_intro_project_manager.png
+
+At the top of the window, there is another tab named "Templates". You can search
+for demo projects in the open-source asset library, which includes many projects
+developed by the community.
+
+.. seealso:: To learn the project manager's ins and outs, read
+             :ref:`doc_project_manager`.
+
+.. image:: img/editor_intro_project_templates.png
+
+You can also change the editor's language using the drop-down menu to the right
+of the engine's version in the window's top-right corner. By default, it is in
+English (EN).
+
+.. image:: img/editor_intro_language.png
+
+First look at Godot's editor
+----------------------------
+
+When you open a new or an existing project, the editor's interface appears.
+Let's look at its main areas.
+
+.. image:: img/editor_intro_editor_empty.png
+
+By default, it features **menus**, **workspaces**, and playtest buttons along
+the window's top edge.
+
+.. image:: img/editor_intro_top_menus.png
+
+In the center is the **viewport** with its **toolbar** at the top, where you'll
+find tools to move, scale, or lock the scene's nodes.
+
+.. image:: img/editor_intro_3d_viewport.png
+
+On either side of the viewport sit the **docks**. And at the bottom of the
+window lies the **bottom panel**.
+
+The toolbar changes based on the context and selected node. Here is the 2D toolbar.
+
+.. image:: img/editor_intro_toolbar_2d.png
+
+Below is the 3D one.
+
+.. image:: img/editor_intro_toolbar_3d.png
+
+Let's look at the docks. The **FileSystem** dock lists your project files, be it
+scripts, images, audio samples, and more.
+
+.. image:: img/editor_intro_filesystem_dock.png
+
+The **Scene** dock lists the active scene's nodes.
+
+.. image:: img/editor_intro_scene_dock.png
+
+The **Inspector** allows you to edit the properties of a selected node.
+
+.. image:: img/editor_intro_inspector_dock.png
+
+The **bottom panel**, situated below the viewport, is the host for the debug
+console, the animation editor, the audio mixer, and more. They can take precious
+space, that's why they're folded by default.
+
+.. image:: img/editor_intro_bottom_panels.png
+
+When you click on one, it expands vertically. Below, you can see the animation editor opened.
+
+.. image:: img/editor_intro_bottom_panel_animation.png
+
+The four workspaces
```

Comment:
These are called "main screens" elsewhere in the documentation. I'm not against "workspaces", but I think we should stick to one name and use it consistently in the documentation.